tests/test_ecephys/test_tools_spikeinterface.py

```diff
@@ -86,15 +75,12 @@ def test_write_as_lfp(self):
         assert "ElectricalSeriesLFP" in lfp_container.electrical_series
 
         electrical_series = lfp_container.electrical_series["ElectricalSeriesLFP"]
-        extracted_data = electrical_series.data[:]
+        extracted_data = np.vstack([x.data for x in electrical_series.data])
```

Comment:
What was the reason for this series of changes?

Reply:
When you wrap a `DataChunkIterator` object in a `DataIO` object, you can use `[:]` to access the data, however if you are trying to access the data directly from a `DataChunkIterator` object, you need to iterate through it.